django_tasks/checks.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, List, Sequence
+
+from django.apps.config import AppConfig
+from django.core.checks.messages import CheckMessage
+
+from django_tasks import tasks
+
+
+def check_tasks(
+    app_configs: Sequence[AppConfig] = None, **kwargs: Any
+) -> List[CheckMessage]:
+    """Checks all registered task backends."""
+
+    errors = []
+    for backend in tasks.all():
```

Comment:
Suggesiton: I _believe_ checks can just use an iterable, so we can probably make this simpler:

```python
for backend in tasks.all():
    yield from backend.check()
```